resources/openapi/api-intro.md

```diff
@@ -26,7 +12,3 @@ Create keys to authenticate programmatic requests to your Metabase. See [API key
 ## API changelog
 
 For breaking changes, see the API [changelog](./developers-guide/api-changelog.md).
-
-## API endpoints
-
-_* indicates endpoints used for features available on [paid plans](https://www.metabase.com/pricing)._
```

Comment:
Hmm I guess we don't have this anymore. But to be fair this was never super useful because which endpoints were available depended on what type of paid plan you had.

Reply:
We can have the endpoint docstring automatically include a "only available in paid plans with 'sandboxing' or something like that"? What makes sense here?

Reply:
Some kind of paid tag makes sense here. "Only available on [Pro and Enterprise plans](https://www.metabase.com/pricing/)."